extensions/blocks/podcast-player/style.scss

```diff
@@ -331,6 +332,8 @@ $player-background: transparent;
 
 	.mejs-controls {
 		position: static;
+		// Lines up the mejs player padding with our wrapper spacing
+		padding: 0 ($player-grid-spacing - 7px) 0 ($player-grid-spacing - 9px);
```

Comment:
Does the wrapper use these magic numbers?

Reply:
No, they use a set `10px` margin, but that's to the _edge of the play/pause button_. We don't have a background wrapping the full button, so we're trying to line-up the edge of the icon within a button. Similar for the sound level. The sound level is a `width: 50px;` set value that doesn't span the full width of its own container, so there isn't really a number to calculate off of. Just trying to visually line things up.